### DISCUSSION TITLE
WAF support

### DISCUSSION THREAD

**Discussion by denzuko** [nginx-proxy/nginx-proxy#2491] [Ideas]
Use case:
   - As a user I would like to ensure that services are protected so that we can maintain high availability.

Acceptance Criteria:
   - should have naxsi enabled in nginx.conf

**Comment by elico** [comment]
I have seen couple containers which implement nginx with waf:
* https://github.com/vlche/docker-nginx-waf
* https://github.com/carlosdg/NginxReverseProxyWithModsecurity

And the docs:
* https://docs.nginx.com/nginx-app-protect-waf/v4/configuration-guide/configuration/

Maybe it can be helpful.
